### PR TITLE
cluster-operator 0.23.9 patch version for azure

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward giantswarm/giantswarm#10211

Creating new patch release which would include cluster-operator 0.23.9. 

- 11.3.1

### CODE REVIEW

**Comment by cokiengchiara** [comment]
@tomahawk28 are you still working on this? :) Let me know if I could help w release notes. Will just copy from the AWS releases.

**Comment by tomahawk28** [comment]
I think @giantswarm/team-celestial is working on this, so I was about to close this PR, but can you communicate with them?

**Comment by cokiengchiara** [comment]
> I think @giantswarm/team-celestial is working on this, so I was about to close this PR, but can you communicate with them?

ok doing that here https://github.com/giantswarm/giantswarm/issues/10871

File: azure.yaml [hunk 1]
```diff
@@ -3,9 +3,63 @@ kind: Release
...
+      componentVersion: 1.9.5
+      version: 1.0.5
+    - name: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - name: net-exporter
+      version: 1.7.0
+    - name: nginx-ingress-controller
+      componentVersion: 0.30.0
+      version: 1.6.9
```
Comment (sslavic) [inline azure.yaml:31]: Please hold with this one, 0.23.9 cluster-operator seems to expect that nginx 1.6.10 is being used, @whites11 correct?
Extra context see: https://github.com/giantswarm/cluster-operator/pull/1050

Comment (MartaD) [reply azure.yaml:31]: I think this PR can be closed, I've added this change to your PR Stevo: https://github.com/giantswarm/releases/pull/269 
it's also renamed from 11.3.1 to 11.4.0.

Comment (sslavic) [reply azure.yaml:31]: Cool, thanks for heads up. Will close this one then. @tomahawk28 feel free to reopen if you think differently

**Pull request closed without merging**